### DISCUSSION TITLE
How to disable context menu when Right-click to Pan camera?

### DISCUSSION THREAD

**Discussion by wawa2016** [google/model-viewer#1843] [Q&A] [answered]
Hi, I am getting a very annoying behaviour, when right-clicking to pan and re-leasing the mouse button I get a context menu https://modelviewer.dev/examples/stagingandcameras/#panning
Is there any chance to disable turning on the context menu from the panning camera script please?

**Comment by elalish** [comment, marked as answer]
Indeed, that is annoying! This looks pretty promising: https://stackoverflow.com/a/16987577/7852087
In fact I should probably just add this to the example.

**Reply by wawa2016** [reply]
Thanks it worked perfectly!
